README.md

```diff
@@ -177,6 +188,11 @@ running `rbenv rehash`. Likewise, if you have multiple copies of the
 vagrant gem installed, you'll need to clean up all versions. This
 recipe won't support such craziness :-).
 
+# Notes about specific plugins
+*  vagrant-libvirt. Installing this plugin has required setting environment
+variables on ubuntu system. Adding env CONFIGURE_ARGS: 'with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib'
+to the vagrant_plugin resource properties has worked.
+
```

Comment:
Does it make sense to add an usage example?

It wasn't clear without reading the resource that `env` is the property, I _think_ this is intended usage?

```
vagrant_plugin 'vagrant-aws' do
  user 'donuts'
  env CONFIGURE_ARGS: 'with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib'
  action :install
end
```

Is this always going to be the same on Ubuntu?  Are there scenarios where this is not required on Ubuntu?

Would it make sense instead of adding this as a property have a constant?  Or to make it easier on users set the default for ubuntu, maybe something to the effect of:

```
# resources/plugin.rb
property :env, [Hash, nil], default: lazy {
  if platform?('ubuntu')
    CONFIGURE_ARGS: 'with-libvirt-include=/usr/include/libvirt with-libvirt-lib=/usr/lib'
  else
    nil
  end
}
```

I think it's ok as-is, but this seems like a property that most people won't want to know about unless they need to know about it.

Reply:
The env property settings in the discussion only apply to the vagrant-libvirt plugin. Those aren't general ubuntu settings.  Someone installing that plugin also has to install the package prereqs.  I don't want to or know how to maintain a table of the prereqs for each plugin.  If we needed something for all the plugins for ubuntu I'd do somthing like you suggest.

Reply:
Gotcha!  Ok, that was the piece I was missing, this is specifically for `vagrant-libvirt`.

Totally makes sense, I think that's reasonable.